dependencies/maven/BUILD

```diff
@@ -14,4 +14,11 @@
 #
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
-#
\ No newline at end of file
+#
+
+genrule(
+    name = "deployment_rules",
+    srcs = ["@graknlabs_rules_deployment//maven/templates:rules.bzl", "//:deployment.properties"],
+    cmd = "MAVEN_PACKAGES=`grep 'maven.packages' $(location //:deployment.properties) | cut -d '=' -f 2`; sed -e s/{maven_packages}/$$MAVEN_PACKAGES/g $(location @graknlabs_rules_deployment//maven/templates:rules.bzl) >> $@",
+    outs = ["rules.bzl"]
```

Comment:
This will be the place where we provide the `deployment.properties` file as input to generate `rules.bzl`.

Reply:
Temporarily we will still need to provide `//:VERSION` file as input too, until we move `version` as a property into `deployment.properties`